src/components/playback/playbackmanager.js

```diff
@@ -1895,6 +1898,12 @@ class PlaybackManager {
                         }
                     }
 
+                    for (let i = 0; i < episodesResult.Items.length; i++) {
+                        if (i > 0) {
+                            episodesResult.Items[i].UserData.PlaybackPositionTicks = 0;
+                        }
+                    }
```

Comment:
So this is ensuring that all episodes in the queue start at the beginning?

Reply:
That is the idea, but in testing it seems that hitting the season/series play button does not keep this information, at the end of an episode it seems to just load the next one. In the other procedure `isFirstItem` seems to always be true.